src/components/AvatarList.astro

```diff
@@ -1,10 +1,56 @@
+---
+export let path;
+// fetch all commits for just this page's path
+let url = `https://api.github.com/repos/snowpackjs/astro-docs/commits?path=${path}`;
+let token = import.meta.env.SNOWPACK_PUBLIC_GITHUB_TOKEN ?? throw Error('You might not have "SNOWPACK_PUBLIC_GITHUB_TOKEN added for escaping rate-limiting.');
```

Comment:
can we leave this as optional? It would be a big point of friction to need to generate an access token just to start up dev.

If it's a concern that this throws an error when rate limited: we can catch that error, ignore it, and then return an empty avatar list. This would only happen in dev, but then still let you provide your own token if you need to

Reply:
That certainly sounds reasonable